Runtime/PostProcessing/Shaders/CRT.shader

```diff
@@ -139,6 +150,91 @@ Shader "Hidden/HauntedPS1/CRT"
+        float s = sin(carrier_phase);
+        float c = cos(carrier_phase);
+
+        float modulated = I * s + Q * c;
+        float3 premultiplied = float3(Y, 2 * s * modulated, 2 * c * modulated);
+        
+        return premultiplied;
+    }
+
+    float Gaussian(int positionX, int halfRange)
+    {
+        return exp2(-.5 * ((float)positionX / halfRange * (float)positionX / halfRange));
+    }
+
+    float3 ComputeGaussianInYIQ(float2 positionFramebufferNDC)
+    {
+        float3 colorTotal = 0;
+        float weightTotal = 0.0;
+        
+        float2 positionFramebufferCenterPixels = positionFramebufferNDC * _ScreenSizeRasterizationRTScaled.xy;
+        int halfRange = 3;
+        for (int x = -halfRange; x <= halfRange; ++x)
+        {
+            // Convert from framebuffer normalized position to pixel position
+            float2 positionFramebufferCurrentPixels = positionFramebufferCenterPixels + float2(x * (_NTSCKernelWidthRatio * _NTSCHorizontalCarrierFrequency), 0);
+            float2 positionFramebufferCurrentNDC = positionFramebufferCurrentPixels * _ScreenSizeRasterizationRTScaled.zw;
+            
+            positionFramebufferCurrentNDC = ClampRasterizationRTUV(positionFramebufferCurrentNDC);
+            float3 colorCurrent = FCCYIQFromSRGB(FetchFrameBuffer(positionFramebufferCurrentNDC).rgb);
+
+            // Use the original offset value in order to keep it the same distance across resolutions
+            colorCurrent = QuadratureAmplitudeModulation(colorCurrent, positionFramebufferCurrentPixels);
+
+            // Fetch the Gauss weight at the current x offset position
+            float weightCurrent = Gaussian(x, halfRange);
+
+            colorTotal += colorCurrent * weightCurrent;
+            weightTotal += weightCurrent;
+        }
+
+        // After loop.
+        colorTotal /= weightTotal;        
+        
+        return colorTotal;
+    }
+    
+    float4 ComputeAnalogSignal(float2 positionFramebufferNDC)
+    {
+        float4 time = _TimeUnscaled;
+        if (_NTSCFlickerUseTimeScale == 1) time = _Time;
+        float2 offsetFlicker = (_NTSCFlickerScaleX * float2(sign(sin(time.y * (60 * _NTSCFlickerPercent)) * sign(sin(positionFramebufferNDC.y * _ScreenSizeRasterizationRTScaled.y * _NTSCFlickerScaleY))), 0))
+            * _ScreenSizeRasterizationRTScaled.zw;
+        positionFramebufferNDC += offsetFlicker;
+        
+        positionFramebufferNDC = ClampRasterizationRTUV(positionFramebufferNDC);
+        float4 color = FetchFrameBuffer(positionFramebufferNDC);
+
+        // Convert color space to FCCYIQ
+        color.rgb = FCCYIQFromSRGB(color.rgb);
+
+        // color = QuadratureAmplitudeModulation(color, positionFB, positionSS);
+
+        float oldY = color.r;
+        
+        // Compute the Gaussian effect
+        color.rgb = ComputeGaussianInYIQ(positionFramebufferNDC);
+
+        color.r = oldY + (_NTSCSharpenPercent * (oldY - color.r));
```

Comment:
Rename this to _NTSCSharpenRatio or just _NTSCSharpness. This is not a percent, it's a [0.0, 1.0] ratio from what I can tell with the code.